bool crosswalk_detected
float32 crosswalk_distance
bool traffic_light_detected
int32 light_type  # 0: None, 1: Red, 2: Green
